backend/server.js

```diff
@@ -0,0 +1,126 @@
+const express = require('express')
+
+
+const postgresql= require(  "./postgresql");
+
+
+const app = express();
+
+let bodyParser = require('body-parser');
+const port = 3000
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+
+
+
+app.get("/check/:poll_name", async (req, res) => {
```

Comment:
Should be `app.get("/check/:guild_id/:poll_name")`. Again, please see #8 .

Reply:
isnt this end point for checking whether the pollname was unique or not ?